app/scripts/modules/kubernetes/src/v2/pipelines/stages/deployManifest/deployManifestConfig.html

```diff
@@ -37,14 +37,16 @@ <h4>Manifest Configuration</h4>
                              metadata="ctrl.metadata" change="ctrl.change()">
   </kubernetes-manifest-entry>
 
-  <expected-artifact-selector ng-if="ctrl.$scope.stage.source === ctrl.artifactSource"
-                              command="ctrl.$scope.stage"
-                              id-field="manifestArtifactId"
-                              account-field="manifestArtifactAccount"
-                              accounts="ctrl.metadata.backingData.artifactAccounts"
-                              expected-artifacts="ctrl.expectedArtifacts"
-                              help-field-key="kubernetes.manifest.expectedArtifact">
-  </expected-artifact-selector>
+  <div class="container-fluid form-horizontal">
```

Comment:
we'll end up with an empty `<div>` in the DOM if `ctrl.$scope.stage.source !== ctrl.artifactSource`.  Is this intentional or should the `ng-if` from `expected-artifact-selector` move up to this div?

Reply:
Good point, I'll fix that